snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -25,37 +25,97 @@ func snapshot(name: String, waitForLoadingIndicator: Bool = true) {
     Snapshot.snapshot(name, waitForLoadingIndicator: waitForLoadingIndicator)
 }
 
-class Snapshot: NSObject {
+func changeKeyboard(app: XCUIApplication, systemLocale: String) {
```

Comment:
How about `func changeKeyboard(app: XCUIApplication, systemLocale: String = "en")` so you can drop the short version below?